packages/ws/src/strategies/sharding/SimpleShardingStrategy.ts

```diff
@@ -0,0 +1,68 @@
+import { Collection } from '@discordjs/collection';
+import type { GatewaySendPayload } from 'discord-api-types/v10';
+import type { IShardingStrategy } from './IShardingStrategy';
+import { IdentifyThrottler } from '../../utils/IdentifyThrottler';
+import type { WebSocketManager } from '../../ws/WebSocketManager';
+import { WebSocketShard, WebSocketShardDestroyOptions, WebSocketShardEvents } from '../../ws/WebSocketShard';
+import { managerToFetchingStrategyOptions } from '../context/IContextFetchingStrategy';
+import { SimpleContextFetchingStrategy } from '../context/SimpleContextFetchingStrategy';
+
+/**
+ * Simple strategy that just spawns shards in the current process
+ */
+export class SimpleShardingStrategy implements IShardingStrategy {
+	private readonly manager: WebSocketManager;
+	private readonly shards = new Collection<number, WebSocketShard>();
+
+	private readonly throttler: IdentifyThrottler;
+
+	public constructor(manager: WebSocketManager) {
+		this.manager = manager;
+		this.throttler = new IdentifyThrottler(manager);
+	}
+
+	public async spawn(shardIds: number[]) {
+		await this.destroy({ reason: 'User is adjusting their shards' });
```

Comment:
Hmmm...so if users call .spawn twice it'll respawn? I don't think thats needed

Reply:
Keep in mind this is the strategy's `spawn`, it's only referenced here:
https://github.com/discordjs/discord.js/blob/71117182ac8fe2b7de1b34d6b58a32fb6a270d6d/packages/ws/src/ws/WebSocketManager.ts#L213-L221

That was there before I added that first `destroy()` call at the top, will adjust.